.github/workflows/release.yaml

```diff
@@ -1,21 +1,45 @@
-name: "Prepare the release"
+name: "Create the release"
 on:
   push:
-    tags: [ 'v*' ]
+    branches:
+      - 'release/**'
```

Comment:
does this mean we should start using release branches?

Reply:
No, it means we can if we need to. Without this entry we wouldn't have the option.

Reply:
thanks for the clarification, ill wait on pavol's thoughts before merging.